src/current/v25.1/changefeed-monitoring-guide.md

```diff
@@ -0,0 +1,168 @@
+
+## Pipeline section metrics
+
+### Rangefeed
+
+#### Buffer memory usage
+
+- Metric: `changefeed.buffer_entries.allocated_mem.rangefeed`
+- Description: The current quota pool memory allocation between the rangefeed and the KV feed.
+- Impact: High memory usage may indicate backpressure.
+- Supported versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
+
+#### Rangefeed buffer latency
+
+- Metrics: 
+    - `changefeed.stage.rangefeed_buffer_checkpoint.latency` (count/sum/bucket)
+    - `changefeed.stage.rangefeed_buffer_value.latency` (count/sum/bucket)
+- Description: Latency within the rangefeed section of the changefeed pipeline.
+- Impact: Indicates potential scanning or changefeed catch-up issues.
+- Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels).
+- Supported Versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
+
+#### KV feed
+
+- Metric: `changefeed.stage.kv_feed_wait_for_table_event.latency` (count/sum/bucket)
+- Description: Latency within the [processing section](#processing-aggregation-and-encoding) of the changefeed pipeline.
+- Impact: Potential bottlenecks in encoding, batching, sending data, or at the [sink]({% link {{ page.version.version }}/changefeed-sinks.md %}). Use this metric in conjunction with the [downstream delivery metrics](#downstream-delivery).
+- Supported Versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
+
+### Processing — aggregation and encoding
+
+#### Aggregator buffer health
+
+- Metrics:
+    - `changefeed.buffer_entries.allocated_mem.aggregator`
+    - `changefeed.stage.kv_feed_buffer.latency` (count/sum/bucket)
+- Description: The current quota pool memory allocation between the KV feed and the sink. Latency within the processing section of the changefeed pipeline.
+- Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels).
+- Supported Versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
+
+#### Encoding performance
+
+- Metric: `changefeed.stage.encode.latency` (count/sum/bucket)
+- Description: Latency encoding data within the processing section of the changefeed pipeline.
+- Impact: High encoding latency can create a bottleneck for the entire changefeed pipeline.
+- Use with [metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels).
+- Supported Versions: v23.2.13+, v24.1.6+, v24.2.4+, v24.3.0+
+
+### Sink
+
+#### Sink performance 
+
+{{site.data.alerts.callout_info}}
+This metric is supported for [webhook]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink), [Google Cloud Pub/Sub]({% link {{ page.version.version }}/changefeed-sinks.md %}#google-cloud-pub-sub), and the latest version of the CockroachDB [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka) sink, which is the [default from v24.2.1]({% link v24.2/changefeed-sinks.md %}#kafka).
```

Comment:
that the kafka sink is the default since v24.2.1 is already documented at the link target